R/results_parsers.R

```diff
@@ -41,7 +41,7 @@ parse_errors_to_df <- function(data) {
   attr(data, error_class) %>%
     purrr::map_df(
       ~ tibble::tibble(
-        assertion.id = .$assertion.id,
+        assertion.id = generate_id(), # TODO(pawel): .$assertion.id,  Error: Column `assertion.id` not found in `.data`
```

Comment:
this should be fixed now, do you know how to recreate the issue?

Reply:
Yes, now it should work correctly. I removed this TODO

Reply:
I mean this should work fine with having .assertion.id. I'll double check it.